examples/CRISP/scripts/tasks/cli.sh

```
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+(cd ./apps/server && ./target/debug/cli $@)
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)